docs/sources/vendor/Barracuda/barracuda.md

```diff
@@ -0,0 +1,27 @@
+# Barracuda
+
+
+## Key facts
+
+* MSG Format based filter
+* Legacy BSD Format default port 514
+
+## Links
+
+| Ref            | Link                                                                                                    |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| Splunk Add-on  | None                                                               |
+| Product Manual | unknown |
+
+## Sourcetypes
+
+| sourcetype      | notes                                                                                                   |
+|-----------------|---------------------------------------------------------------------------------------------------------|
+|barracuda:web:firewall   |  none  |
+
+## Sourcetype and Index Configuration
+
+| key             | sourcetype     | index  | notes          |
+|-----------------|----------------|--------|----------------|
+| Barracuda       |  barracuda:web:firewall  | netfw  | None     |
```

Comment:
the key is "vendor_product_class" where _class is optional so the key here should be barracuda_waf